pkgs/applications/networking/instant-messengers/rambox/rambox.nix

```diff
@@ -0,0 +1,32 @@
+{ pname, version, src, meta, desktopName ? "Rambox" }:
+
+{ appimageTools, stdenv, lib, fetchurl, makeDesktopItem }:
+
+let
+  name = "${pname}-${version}";
+
+  desktopItem = (makeDesktopItem {
+    inherit desktopName;
+    name = pname;
+    exec = pname;
+    icon = pname;
+    type = "Application";
+    categories = "Network;";
+  });
+
+  appimageContents = appimageTools.extractType2 {
+    inherit name src;
+  };
+in appimageTools.wrapType2 rec {
+  inherit name src meta;
+
+  extraPkgs = pkgs: with pkgs; [ xdg_utils ];
```

Comment:
Opening links in browser are not working (tested in `rambox`), but I don't know if they worked in `master` or not. The only thing that the old derivation had that seems related is `xdg_utils`, but even adding it here doesn't work it seems.

Some help would be appreciated.

Reply:
We could also merge it as is and open an issue to investigate it later.

Reply:
Just confirming that it works in `master`.

Reply:
Just saw this is unnecessary (it is in the default packages from `wrapType2`). Removing it.